src/interface/src/styleguide/notes-sidebar/notes-sidebar.component.ts

```diff
@@ -0,0 +1,120 @@
+import { Component, Input, OnInit, Output, EventEmitter } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { DeleteNoteDialogComponent } from '../../app/plan/delete-note-dialog/delete-note-dialog.component';
+import { take } from 'rxjs';
+import { Note, NotesModelName, BaseNotesService } from '@services';
+import { SNACK_ERROR_CONFIG, SNACK_NOTICE_CONFIG } from '@shared';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatIconModule } from '@angular/material/icon';
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { FormsModule } from '@angular/forms';
+import { CommonModule } from '@angular/common';
+import { MatInputModule } from '@angular/material/input';
+
+@Component({
+  standalone: true,
+  selector: 'sg-notes-sidebar',
+  templateUrl: './notes-sidebar.component.html',
+  styleUrls: ['./notes-sidebar.component.scss'],
+  imports: [
+    MatMenuModule,
+    CommonModule,
+    MatIconModule,
+    MatProgressSpinnerModule,
+    MatFormFieldModule,
+    FormsModule,
+    MatInputModule,
+  ],
+})
+export class NotesSidebarComponent implements OnInit {
+  constructor(
+    private notesService: BaseNotesService,
```

Comment:
I dont think this is right, we don't wanna tie services with the styleguide components.
You can probably use @inputs and @outputs instead

Reply:
yep, this is the main reason this is still in draft.